### ISSUE TITLE
--skeleton flag doesn't work

### ISSUE THREAD

**Issue by balupton** [docpad/docpad#942]
```
> docpad run --skeleton h5bp

/Users/balupton/Projects/docpad/node_modules/commander/index.js:284
      args.push(self);
           ^
TypeError: undefined is not a function
  at Command.listener (/Users/balupton/Projects/docpad/node_modules/commander/index.js:284:12)
  at Command.emit (events.js:129:20)
  at Command.parseOptions (/Users/balupton/Projects/docpad/node_modules/commander/index.js:616:14)
  at Command.parse (/Users/balupton/Projects/docpad/node_modules/commander/index.js:424:21)
  at ConsoleInterface.start (/Users/balupton/Projects/docpad/out/lib/interfaces/console.js:97:22)
  at ConsoleInterface.start (/Users/balupton/Projects/docpad/out/lib/interfaces/console.js:4:59)
  at /Users/balupton/Projects/docpad/out/bin/docpad.js:42:33
  at /Users/balupton/Projects/docpad/out/lib/interfaces/console.js:90:16
  at TaskGroup.<anonymous> (/Users/balupton/Projects/docpad/out/lib/docpad.js:1259:16)
  at TaskGroup.g (events.js:199:16)
  at TaskGroup.emit (events.js:110:17)
  at TaskGroup.<anonymous> (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:53:28)
  at TaskGroup.emit (events.js:110:17)
  at TaskGroup.complete (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:952:14)
  at TaskGroup.exit (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:1096:12)
  at TaskGroup.fire (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:1054:16)
  at TaskGroup.itemCompletionCallback (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:1047:12)
  at Task.<anonymous> (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:746:42)
  at Task.g (events.js:199:16)
  at Task.emit (events.js:129:20)
  at Task.<anonymous> (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:53:28)
  at Task.emit (events.js:129:20)
  at Task.complete (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:307:19)
  at Task.exit (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:289:14)
  at complete (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:376:26)
  at DocPad.<anonymous> (/Users/balupton/Projects/docpad/out/lib/docpad.js:1071:20)
  at ambi (/Users/balupton/Projects/docpad/node_modules/ambi/out/lib/ambi.js:55:18)
  at Task.<anonymous> (/Users/balupton/Projects/docpad/node_modules/event-emitter-grouped/out/lib/event-emitter-grouped.js:45:23)
  at ambi (/Users/balupton/Projects/docpad/node_modules/ambi/out/lib/ambi.js:55:18)
  at Domain.fireMethod (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:385:23)
  at Domain.run (domain.js:197:16)
  at Task.fire (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:423:27)
  at Immediate._onImmediate (/Users/balupton/Projects/docpad/node_modules/taskgroup/out/lib/taskgroup.js:440:26)
  at processImmediate [as _immediateCallback] (timers.js:358:17)
```

No clue why.

**Comment by Greduan** [comment]
Damn that error. It means that variable didn't get defined as an array at some point before using `.push()`.

**Comment by balupton** [comment]
Yep, fixed in v6.74.0. It was due to commander not support actions that have the same name as an option. Solution was to just get rid of the action, as we never really used it.
